presto-kafka/pom.xml

```diff
@@ -68,14 +68,20 @@
         </dependency>
 
         <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
+            <version>${dep.kafka-clients.version}</version>
```

Comment:
Add this to the `dependencyManagement` section in the root POM, then remove the version here. In almost all cases, versions should only be in the root POM, ensuring versions are consistent.

(Technically, this is not necessary here, because this dependency is only used in this one module, and different plugins can actually have different versions of the same library due to class loader isolation. But we follow the rule as much as possible so it's easy for everyone to understand and not wonder why there are exceptions.)